### DISCUSSION TITLE
Is there a preferred way to manage CSS

### DISCUSSION THREAD

**Discussion by backspaces** [vanjs-org/van#41] [General]
I was wondering if there is a good fit between vanjs and similarly simplified css tools?
I've been following Stephanie Eckles at https://thinkdobecreate.com/ and her https://smolcss.dev/ snippets are compelling. Simple and no "framework".

**Comment by Tao-VanJS** [comment]
VanJS can be freely combined with any CSS templates. VanJS is unopininated regarding which template to use. Thus you can choose the one based on your preference.

**Comment by sysmat** [comment]
how to reactive change class in van tags?

**Reply by sirenkovladd** [reply]
you can take a look at the example with [diff](https://vanjs.org/demo#diff)
this is one of the examples of how you can reactively change a class

**Reply by sysmat** [reply]
ok, then when to use it as func?

```typescript
{ class: () => (counter.val > 2 ? 'text-2xl' : 'text-xl') },
```

**Reply by Tao-VanJS** [reply]
Hi @sysmat,

If you want the `class` to be reactive to VanJS states, you can wrap the value with a binding function.
